.github/workflows/lint.yml

```diff
@@ -0,0 +1,28 @@
+name: lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install VapourSynth
+        uses: Irrational-Encoding-Wizardry/install-vapoursynth-action@v0.1
+        with:
+          version: 50
+          cache: true
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          pip install -r requirements-dev.txt
+          pip install .
+          pip install -U git+https://github.com/Irrational-Encoding-Wizardry/vsutil.git@stux-switch-to-package#egg=vsutil
+      - name: Running flake8
+        run: flake8 lvsfunc --count --ignore=E501,W503,E226 --show-source --statistics
+      - name: Running mypy
+        run: mypy -p lvsfunc --show-column-numbers --show-error-codes --hide-error-context --no-error-summary
```

Comment:
I suggest adding this configuration in config files inside the repo instead of in the CI commands, so they can easily be reproduced locally through just `flake8 lvsfunc` or linter integration tools.